### PR TITLE
Check if packetid is invalid and skipbytes in PACKET_NOT_READ_TO_END Exception.

### PR DESCRIPTION
**Pull request by PermisosDev**
.

### CODE REVIEW

**Comment by linsaftw** [comment]
I saw you check `packetid > 255`. What if Mojang adds new packet types and exceed this number? I think you have to remove the `packetid > 255` check.

**Review by linsaftw — CHANGES_REQUESTED** [review]
I saw you check packetid > 255. What if Mojang adds new packet types and exceed this number? I think you have to remove the packetid > 255 check.

**Comment by PermisosDev** [comment]
> I saw you check packetid > 255. What if Mojang adds new packet types and exceed this number? I think you have to remove the packetid > 255 check.

in bungeecord the max packetid is 0xFF (255 in hexadecimal). if mojang adds new packet, that value can be changed in the upstream (i think).

File: Waterfall-Proxy-Patches/0026-Add-skip-bytes-on-PACKET_NOT_READ_TO_END-and-changed.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+diff --git a/protocol/src/main/java/net/md_5/bungee/protocol/MinecraftDecoder.java b/protocol/src/main/java/net/md_5/bungee/protocol/MinecraftDecoder.java
+index b3de7a7b..9b7934b8 100644
+--- a/protocol/src/main/java/net/md_5/bungee/protocol/MinecraftDecoder.java
++++ b/protocol/src/main/java/net/md_5/bungee/protocol/MinecraftDecoder.java
+@@ -66,7 +66,7 @@ public class MinecraftDecoder extends MessageToMessageDecoder<ByteBuf>
+             int packetId = DefinedPacket.readVarInt( in );
+             packetTypeInfo = packetId;
+             // FlameCord Start - Check if packetId is invalid.
+-            if (packetId < 0 || packetId > 255)
++            if (packetId < 0 || packetId > Protocol.MAX_PACKET_ID)
```
Comment (PermisosDev) [inline Waterfall-Proxy-Patches/0026-Add-skip-bytes-on-PACKET_NOT_READ_TO_END-and-changed.patch:17]: Changed. not 255 anymore.

**Pull request merged**